modules/webapp/src/main/elm/Data/Language.elm

```diff
@@ -90,6 +91,9 @@ fromString str =
     else if str == "jpn" || str == "ja" || str == "japanese" then
         Just Japanese
 
+    else if str == "jpn_vert" || str == "ja_vert" || str == "jpnvert" then
+        Just Japanese
```

Comment:
Shouldn't this read `JpnVert`?

Reply:
Hi @tenpai-git not sure if you saw that comment? From what I understand this should perhaps be `JpnVert` no?

Reply:
I'm not sure the case was created.

Does this refer in anyway to `modules/analysis/src/test/scala/docspell/analysis/date/DateFindTest.scala`, which should be the same for Japanese and JpnVert? I didn't include a test for JpnVert so I thought it was safer just to list Japanese but I can change to JpnVert if there's nothing broken by that.

Reply:
Oh sorry, for some reason I didn't see your first comment here 😕 

This `Language.elm` file is the client version of the language and is used in dropdowns etc. It's iso3 code may be transferred to the server. The other way around, a value might be constructed from a iso3 string (that was received). In this case it should probably return the respective value and not another one. So if the server sends `jpn_vert` (our psoudo language code), the ui should probably present the `JpnVert` value, I think.

Reply:
I understand, thank you for clearing that up. I will change it to `JpnVert`.